packages/flame/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
  - Create default implementations of `RectangleComponent`, `CircleComponent` and `PolygonComponent`
  - Streamlined the argument list for all components extending `PositionComponent`
  - Improved interaction between viewport and isHud components
+ - Calling super-method in `.render()` is now optional
+ - Rendering code that does canvas manipulation is now responsible for saving/restoring the canvas
```

Comment:
I think it is important to note that this changes are only related to the components, not the game.

Reply:
ok, i'll re-phrase it